### ISSUE TITLE
Doesn't work on mac sonoma 14.0

### ISSUE THREAD

**Issue by JosefGvirt** [vitorgalvao/download-media-workflow#8]
### Workflow version

2023.3

### Alfred version

5.1.3

### macOS version

sonoma 14.0

### Debugger output

```alfred_debugger
I've tried to download several YouTube videos using the DV shortcut, and every video does download separate audio and mp4 files. Still, it doesn't stitch them together and eventually, I get the notification 'download failed to try again maybe using DP'. I do have ffmpeg codec and also updated the homebrew using fixed dependencies.
```

### More details

I've tried to download several YouTube videos using the DV shortcut, and every video does download separate audio and mp4 files. Still, it doesn't stitch them together and eventually, I get the notification 'download failed to try again maybe using DP'. I do have ffmpeg codec and also updated the homebrew using fixed dependencies.

**Comment by vitorgalvao** [comment]
Your issue lies somewhere in [yt-dlp](https://github.com/yt-dlp/yt-dlp). The workflow asks it to do the download and whatever you get is 100% dependent on that. Try the download manually in the terminal and report what you get in their issue tracker.

For future issues, please add the correct [debugger output](https://www.alfredapp.com/help/workflows/advanced/debugger/). Without that it’s hard to diagnose issues.

Also, please make the issue title more descriptive and related to the actual error. “Doesn’t work” is generic, more often than not wrong, and makes it hard to parse issues.

Help me help you.

**Comment by JosefGvirt** [comment]
Hey Vitora thanks for the amazingly quick reply. How do I try to run the yt-dlp feature manually on the terminal? @vitorgalvao
